WcaOnRails/app/controllers/api/v0/persons_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Api::V0::PersonsController < Api::V0::ApiController
+  def index
+    persons = Person.current.includes(:user, :ranksSingle, :ranksAverage)
+    if params[:wca_ids].present?
+      persons = persons.where(wca_id: params[:wca_ids].split(','))
```

Comment:
Do you think we should limit this to `Kaminari.config.max_per_page` wca_ids?

Reply:
I don't think it's necessary.
1. I can imagine someone requesting 150 WCA IDs and then using pagination links from response headers to iterate over the pages.
2. Given that browsers put some limit on a URL length, it is not possible to specify a very long list.

Don't have a strong opinion though =)

Reply:
Oh! I didn't realize we're paginating the response. You're absolutely right, this seems fine as is.

Reply:
Just for the record: I discussed some ideas with @jfly offline and we are going to leave it as it is for now.
If we would like to check for non-existing WCA IDs given we would need to retrieve all the corresponding people from the db, and thus loose one advantage of using pagination. For sorting we could either do it in ruby, but it has the same disadvantage. We could also use a `ORDER BY FIELD` trick in the SQL query, but this could slow it down, whereas we don't want to add an overhead  unless necessary.